src/AddIns/Uno.UI.Lottie.Skia/ts/Uno.UI.Lottie.ts

```diff
@@ -0,0 +1,255 @@
+﻿declare const require: any;
+declare const config: any;
+
+namespace Uno.UI {
+	import AnimationData = Lottie.AnimationData;
+
+	export interface LottieAnimationProperties {
+		elementId: number;
+		jsonPath?: string;
+		autoplay: boolean;
+		stretch: string;
+		rate: number;
+	}
+
+	export interface RunningLottieAnimation {
+		animation: Lottie.AnimationItem;
+		properties: LottieAnimationProperties;
+	}
+
+	export class Lottie {
+		private static _player: LottiePlayer;
+		private static _runningAnimations: { [id: number]: RunningLottieAnimation } = {};
+		private static _numberOfFrames: number;
+
+		public static setAnimationProperties(newProperties: LottieAnimationProperties, animationData?: AnimationData): string {
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)